Index: data/CRMEntity.php
===================================================================
--- data/CRMEntity.php	(revisión: 4439)
+++ data/CRMEntity.php	(copia de trabajo)
@@ -196,7 +196,12 @@
 		$upload_file_path = decideFilePath();
 
 		//upload the file in server
+		if (is_uploaded_file($filetmp_name))
 		$upload_status = move_uploaded_file($filetmp_name,$upload_file_path.$current_id."_".$binFile);
+		else {
+			$upload_status = copy($filetmp_name,$upload_file_path.$current_id."_".$binFile);
+			unlink($filetmp_name);
+		}
 
 		$save_file = 'true';
 		//only images are allowed for these modules
